katas/pokerhands/maria/deck.test.js

```diff
@@ -0,0 +1,59 @@
+        const hand7 = [{value:9, suit:'C'}, {value:13, suit:'H'}, {value:9, suit:'S'}, {value:9, suit:'D'}, {value:9, suit:'H'}];
+        const hand8 = [{value:9, suit:'C'}, {value:13, suit:'H'}, {value:10, suit:'S'}, {value:11, suit:'C'}, {value:12, suit:'H'}];
+        const hand9 = [{value:9, suit:'C'}, {value:7, suit:'D'}, {value:7, suit:'S'}, {value:9, suit:'D'}, {value:7, suit:'H'}];
+        const numArray = new Deck();
+        expect(numArray.sortHands(hand1)).toEqual([{value:1, suit:'S'}, {value:2, suit:'H'}, {value:9, suit:'C'}, {value:10, suit:'D'}, {value:13, suit:'H'}]);
+        expect(numArray.sortHands(hand2)).toEqual([{value:1, suit:'C'}, {value:4, suit:'S'}, {value:9, suit:'C'}, {value:13, suit:'H'}, {value:14, suit:'H'}]);
+        expect(numArray.sortHands(hand3)).toEqual([{value:4, suit:'S'}, {value:9, suit:'C'}, {value:13, suit:'H'}, {value:13, suit:'C'}, {value:14, suit:'H'}]);
+        expect(numArray.sortHands(hand4)).toEqual([{value:1, suit:'H'}, {value:1, suit:'C'}, {value:4, suit:'S'}, {value:9, suit:'C'}, {value:9, suit:'H'}]);
+        expect(numArray.sortHands(hand5)).toEqual([{value:1, suit:'C'}, {value:9, suit:'C'}, {value:9, suit:'S'}, {value:9, suit:'H'}, {value:13, suit:'H'}]);
+        expect(numArray.sortHands(hand7)).toEqual([{value:9, suit:'C'}, {value:9, suit:'S'}, {value:9, suit:'D'}, {value:9, suit:'H'}, {value:13, suit:'H'}]);
+        expect(numArray.sortHands(hand8)).toEqual([{value:9, suit:'C'}, {value:10, suit:'S'}, {value:11, suit:'C'}, {value:12, suit:'H'}, {value:13, suit:'H'}]);
+        expect(numArray.sortHands(hand9)).toEqual([{value:7, suit:'D'}, {value:7, suit:'S'}, {value:7, suit:'H'}, {value:9, suit:'C'}, {value:9, suit:'D'}]);
+    });
+    it('should be able to receive array and  return hand win in hight card value', () => {
+        const hand1 = [{value:2, suit:'H'}, {value:10, suit:'D'}, {value:1, suit:'S'}, {value:9, suit:'C'}, {value:13, suit:'D'}];
+        const hand2 = [{value:9, suit:'C'}, {value:13, suit:'H'}, {value:4, suit:'S'}, {value:1, suit:'C'}, {value:14, suit:'H'}];
+        const hand3 = [{value:9, suit:'C'}, {value:13, suit:'H'}, {value:4, suit:'S'}, {value:1, suit:'C'}, {value:11, suit:'H'}];
+        const hand4 = [{value:7, suit:'C'}, {value:13, suit:'D'}, {value:4, suit:'S'}, {value:1, suit:'C'}, {value:14, suit:'S'}];
+        const hand5 = [{value:9, suit:'C'}, {value:13, suit:'D'}, {value:6, suit:'S'}, {value:1, suit:'C'}, {value:14, suit:'S'}];
+        const hand6 = [{value:9, suit:'C'}, {value:13, suit:'D'}, {value:4, suit:'S'}, {value:2, suit:'C'}, {value:14, suit:'S'}];
+        const hand7 = [{value:2, suit:'C'}, {value:3, suit:'D'}, {value:5, suit:'S'}, {value:8, suit:'C'}, {value:13, suit:'S'}];
+        const hand8 = [{value:2, suit:'D'}, {value:3, suit:'D'}, {value:5, suit:'H'}, {value:8, suit:'5'}, {value:13, suit:'S'}];
+        const numArray = new Deck();
+        expect(numArray.hightCard(hand1, hand2)).toEqual({value:14, suit:'H'});
```

Comment:
This works, but is not very clear why the expected result is an object and what actually is `highCard` doing. It can be improved if name could be something like `compareHands` and expected result is `hand 1 wins with high card` or similar, depending on how your logic is implemented